ci-metrics/README.md

```diff
@@ -0,0 +1,171 @@
+														  | using CI publisher |
+														  +--------------------+
+```
+
+The ELK instance is the data store for storing metrics exported from Jenkins jobs with enabled CI publisher. The exported data from Jenkins jobs is harvested via CI Metrics data listener Jenkins job and stored into the ELK instance.
+
+If you want to install your own ELK instance follow the steps in the section ELK instance.
+
+If you do not want to install you own ELK instance skip the sections 'ELK instance' and 'CI Metrics data listener Jenkins job' sections.
+
+
+## ELK instance
+
+### What is ELK
+
+ELK is a combination of Elastic Search, Logstash and Kibana. Elasticsearch is a non relational database. Logstash collects and pushes logs to the Elasticsearch database. Kibana then takes the data and visualizes it in graphical form and it also makes it possible to query the data.
+
+Note that in current design Logstash is not used and the data is directly stored from the CI Metrics data listener Jenkins job into Elasticsearch.
+
+### How to install ELK for CI metrics
+
+Note that the solution requires an RHEL 7 or Centos 7 system. To install an ELK instance on your localhost use the provided Ansible yaml file.
+
+
+1. ####Add EPEL repository
+
+  See [https://fedoraproject.org/wiki/EPEL] for installation intructions
+
+
+1. ####Install git and ansible
+
+  >$ yum -y install git ansible
+
+
+1. ####Get CI Metrics files
+
+  >$ git clone https://github.com/RHQE/platform-ci.git
+
+  >$ cd platform-ci/ci-metrics
+
+
+1. ####Run ansible to install ELK
+
+  >$ ansible-playbook install_elk.yaml
+
+
+1. ####Verify that ELK instance works
+
+    Access the port 80 on your localhost via your web browser and you should see the Kibana configuration page.
+
+  [http://localhost]
+
+
+## ELK Listener Jenkins job
+
+ELK Listener job listens on the CI message bus for messages matching this JMS selector:
+
+```
+(CI_TYPE = 'brew-taskstatechange' AND method = 'build' AND scratch = FALSE) OR (CI_TYPE = 'ci-metricsdata')
+
+```
+
+These messages are stored directly into the ELK instance.
+
+The ELK Listener job requires:
+
+  - an existing ELK instance
+
+  - an existing PDC instance - see https://github.com/product-definition-center/
+
+  - a keytab for autentication to PDC via keberos
+
+### Installation of ELK Listener job
+
+1. ####Get CI Metrics files
+
+  >$ git clone https://github.com/RHQE/platform-ci.git
+
+  >$ cd platform-ci/ci-metrics
+
+1. ####Tweak [ci_listener.yaml](/ci-metrics/ci_listener.yaml)
+
+    Edit the file and:
+
+    - replace LINKTO_ELK_SERVER  with url to an ELK instance
+
+    - replace LINKTO_PDC url to PDC instance
+
+    Also make sure the environment variables KEYTAB and PRINCIPAL are defined in your Jenkins environment with details about your kerberos principal and keytab.
+
+1. ####Enable CI publisher for your Jenkins Job
+
+    To enable the job run jenkins-jobs from the Jenkins Job Builder on the file ci_listener.yaml.
```

Comment:
"To enable the job, run jenkins-jobs..."  Added a comma so I don't read it as "enable the job run"